vendor/github.com/xanzy/go-gitlab/gitlab.go

```diff
@@ -290,6 +290,7 @@ func newClient(options ...ClientOptionFunc) (*Client, error) {
 	c.GroupVariables = &GroupVariablesService{client: c}
 	c.Groups = &GroupsService{client: c}
 	c.InstanceCluster = &InstanceClustersService{client: c}
+	c.InstanceVariables = &InstanceVariablesService{client: c}
```

Comment:
@nicholasklick @armsnyder this was changed manually without a go mod update/vendor, build will break if golang tries to build without the local vendor files. Will PR an update for xanzy/go-gitlab.

Reply:
You're amazing, thank you. 😄 How do we prevent this from occurring in the future? Can we move forward with #461 @roidelapluie, or did you determine that that solution will not work?

Reply:
Removing the vendor folder would prevent this in the future, happy to pul the changes from #461 into #499 if we want to address it in one go.